lib/sync/index.js

```diff
@@ -90,7 +92,7 @@ function invoke(dataset_id, params, callback) {
 
   // Verify that fn param has been passed
   if (!params || !params.fn) {
-    var err = 'no fn parameter provided in params "' + util.inspect(params) + '"';
+    var err = new Error('no fn parameter provided in params "' + util.inspect(params) + '"');
     debugError('[%s] warn %s %j', dataset_id, err, params);
```

Comment:
Using %s results in `{}`, hence this PR https://github.com/feedhenry/fh-mbaas-api/pull/139. %j should only be used for plain old json objects.
e.g.
```javascript
var x = new Error('some message')
console.log(`(%s) (%j)`, x, x)
// (Error: some message) ({})
```